src/www/ui/ui-report-conf.php

```diff
@@ -276,6 +276,9 @@ public function Output()
     if (isset($submitReportConf)) {
       $parms = array();
       $obLicenses = @$_POST["obLicenses"];
+      foreach ($obLicenses as $obTopic => $licArray) {
+        $obLicenses[urldecode($obTopic)] = $licArray;
```

Comment:
Won't changing the data in same array cause duplication?

For example post contains something like `["some+text+%24"] => 23` which after `urldecode()` becomes: `["some+text+%24"] => 23, ["some text $"] => 23` ?

Reply:
Yes in case of multiple array values it will behave like you mentioned. may be unsetting the encoded string would solve this issue. will work on this. thank you.

Reply:
Or even creating a new array?

Reply:
Done